src/components/exercise/ExerciseView.js

```diff
@@ -14,14 +16,22 @@ const ExerciseView = () => {
             .then(res => setCurrentExercise(res.data.data.exercise))
     }, [location.state.id]);
 
+    useEffect(() => {
+        client
+            .get(`/log?exerciseId=2`)
```

Comment:
This exercise ID is hard-coded, was this intended?

Reply:
Sorry, just noticed it. Working on it now. Not intentional